composer.json

```diff
@@ -8,16 +8,16 @@
         "psr/http-client": "^1.0",
         "psr/http-message": "^1.0|^2.0",
         "psr/log": "^1.0|^2.0|^3.0",
-        "guzzlehttp/guzzle": "^7.5",
-        "guzzlehttp/psr7": "^2.0",
-        "webmozart/assert": "^1.11"
+        "webmozart/assert": "^1.11",
+        "psr/http-factory": "^1.0",
```

Comment:
We need real implementation of this interface so `psr/http-factory-implementation` should be required. Same with other PSRs.

Reply:
I just added `psr/http-factory-implementation` and `"psr/http-client-implementation": "^1.0"`.